chat-plugins/mafia.js

```diff
@@ -1910,6 +2265,18 @@ const commands = {
 		},
 		endhelp: [`/mafia end - End the current game of mafia. Requires host % @ * # & ~`],
 
+		showrole: function (target, room, user) {
```

Comment:
I'm actually working on a command like this that also supports other information (roles, modifiers, alignments, themes). Not sure how I feel about parsing the provided role since that can be spammed/abused. My idea was to only show the info for the role/modifier/ect that was selected.

If you want I can take care of this today.